airbyte-db/db-lib/src/test/java/io/airbyte/db/jdbc/TestJdbcUtils.java

```diff
@@ -210,6 +210,42 @@ void testUssSslWithSslSetAndValueIntegerTrue() {
     assertTrue(sslSet);
   }
 
+  @Test
+  void testUseSslWithEmptySslKeyAndSslModeVerifyFull() {
```

Comment:
These tests are good, let's also add an integration test for both `source-mysql-strict-encrypt` and `source-postgres-strict-encrypt` that cover the scenario that was failing for the customer which is when SSH tunnel was set and their `ssl_mode: disabled`

Reply:
> These tests are good, let's also add an integration test for both source-mysql-strict-encrypt and source-postgres-strict-encrypt that cover the scenario that was failing for the customer which is when SSH tunnel was set and their ssl_mode: disabled

@ryankfu i added a couple of tests for postgres and mysql in order to cover such a case that the user encountered (when SSH tunnel was set and ssl_mode: disabled)